.github/workflows/test.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
             pip install .[test]
 
         - name: Run Tests
-          run: pytest -m "not fuzzing" -s --cov
+          run: pytest -m "not fuzzing" -s --cov=src -n auto
```

Comment:
I thought github actions only executes on single core? so this is kind of surprising

Reply:
ape-hardhat has been doing this all along too